.appveyor.yml

```diff
@@ -19,12 +19,12 @@ cache:
 install:
     # Install Rust
     - curl -sSf -o rustup-init.exe https://win.rustup.rs/
-    - rustup-init.exe -y --default-host %TARGET% --default-toolchain stable
+    - rustup-init.exe -y --default-host %TARGET% --default-toolchain beta
     - set PATH=%USERPROFILE%\.cargo\bin;%PATH%
```

Comment:
If this still fails, you could add `rustup default beta` here. And then once this landed, send a follow-up PR that removes it again.

Reply:
Done.